spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher_spec.rb

```diff
@@ -10,39 +10,69 @@
 
   it "tests the refresh" do
     2.times do
-      ems.refresh
-      ems.reload
-
-      assert_ems_counts
-      assert_specific_vm
-      assert_specific_cloud_volume_type
-      assert_specific_cloud_subnet
-      assert_vm_labels
+      mgmt = ems
+      mgmt.refresh
+      mgmt.reload
+
+      assert_ems_counts(mgmt)
```

Comment:
You can reference anything `let()` defines within the same context so this probably doesn't need to be passed in as a parameter unless you want to call `assert_ems_counts` on other objects later

Reply:
Just assuring the same ems is being used throughout. Doing it this way shaved 3 seconds off of the runtime.

Reply:
Wow that's not intuitive, really curious what would cause that

Reply:
Hm idk about that, I ran this version and a version where I removed all of the `mgmt` parameters being passed in and replaced them with just referencing `ems`

This version: `14.2222s` over 5 runs
The other version: `13.8884s` over 5 runs

I don't think it is faster but I think it is negligible.  Up to you but I'd prefer readability over what looks like a negligible difference in speed

Reply:
Originally I was referencing ems inside the different methods. Adding mgmt as a variable to everything was a last minute addition. I tried it on one method and saw it go down by ~1 second. Did another method and a similar result.  Finally I just added it to everything and another second was gone. I don't know where the performance gain was, but I'll take it.

Reply:
It could be that RSpec just doesn't have to do as much.

Reply:
Interesting, okay can you add a comment up [[here]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/195/files#diff-bd2b9ff0484b2f5249c512b2e25de5da51d6926cbe25a2a9ca9faf0ad34828c5R13) indicating that is the reason?